### PR TITLE
Quick start updates

### PR DESCRIPTION
**Pull request by jbusche**
Quick-Start.md tweaks to improve the install experience
## Description

- Remove the clone of the codeflare-sdk, it's not used anywhere else in the doc so not needed.
- Direct apply of the kfdefs rather than curling them and applying them.  (Also, hard to find the codeflare-stack-kfdef.yaml in the existing file, so fixed that.
## How Has This Been Tested?
It's how I've been installing on OpenShift 4.12

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [X] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: Quick-Start.md [hunk 1]
```diff
@@ -19,29 +19,19 @@ This Quick Start guide assumes that you have administrator access to an OpenShif
...
-
 ## Install the CodeFlare Operator
 
 1. First, install the CodeFlare operator from the operatorhub. The default settings for the operator will suffice.
-2. Create the odh-core kfdef. This kfdef can be found in the [odh-manifests repository](https://github.com/opendatahub-io/odh-manifests/blob/master/kfdef/odh-core.yaml)
-3. Create the kfdef in this directory.
-
+2. Apply the odh-core kfdef with this command:
+```bash
+$ oc apply -f https://raw.githubusercontent.com/opendatahub-io/odh-manifests/master/kfdef/odh-core.yaml
```
Comment (tedhtchang) [inline Quick-Start.md]: Don't we need to specify the namespace `-n opendatahub`?

File: Quick-Start.md [hunk 2]
```diff
@@ -19,29 +19,19 @@ This Quick Start guide assumes that you have administrator access to an OpenShif
...
+```
+3. Create the CodeFlare-Stack kfdef with this command:
 ```bash
-$ curl -O https://raw.githubusercontent.com/opendatahub-io/odh-manifests/master/kfdef/odh-core.yaml
-$ oc apply -f odh-core.yaml -n opendatahub
-$ oc apply -f codeflare-stack-kfdef.yaml -n opendatahub  # This should be the same namespace as that for the odh-core deployment
+$ oc apply -f https://raw.githubusercontent.com/opendatahub-io/distributed-workloads/main/codeflare-stack-kfdef.yaml
```
Comment (tedhtchang) [inline Quick-Start.md]: Don't we need to specify the namespace `-n opendatahub`?

**Review by tedhtchang — APPROVED** [review]
/lgtm

File: Quick-Start.md [hunk 3]
```diff
@@ -12,36 +12,30 @@ This stack integrates well with [Open Data Hub](https://opendatahub.io/), and he
...
-3. Create the kfdef in this directory.
-
+1. First, install the CodeFlare operator from the OperatorHub. The default settings for the CodeFlare Operator will suffice.
+2. Create the opendatahub namespace with the following command:
+```bash
+$ oc create ns opendatahub
+```
+3. Apply the odh-core kfdef with this command:
+```bash
+$ oc apply -f https://raw.githubusercontent.com/opendatahub-io/odh-manifests/master/kfdef/odh-core.yaml -n opendatahub
```
Comment (accorvin) [inline Quick-Start.md:31]: I did not know that oc supports applying a file from a URL. This is cool.

**Pull request merged**